### PR TITLE
flesh out acknowledgements

### PR DESCRIPTION
**Pull request by hamogu**

### CODE REVIEW

**Comment by bsipocz** [comment]
This reads very well to me, thanks!

File: main.tex [hunk 1]
```diff
@@ -1258,12 +1258,22 @@ \section{Conclusion}
...
-Who to thank?  numfocus, GSOC, Python Software Foundation
-
-We thank the many web services that make developing and maintaining this package
-feasible, such as GitHub, Travis CI, Appveyor, CircleCI, and Read the Docs.
+\acknowledgments The \astropy community is supported by and makes use
+of a number of organizations and services outside the traditional
+academic community. We thank Google for financing and organizing the
+Google Summer of Code (GSoC) program, that has funded severals
+students per year to work on \astropy related projects over the
+summer. These students ofter turn into long-term contributors. We also
```
Comment (adrn) [inline main.tex]: ofter -> often

File: main.tex [hunk 2]
```diff
@@ -1258,12 +1258,22 @@ \section{Conclusion}
...
+thank NumFOCUS and the Python Software Foundation for financial
+support for various needs. Within the academic community, we thank
```
Comment (adrn) [inline main.tex]: remove "for various needs" (I don't think it is needed)

File: main.tex [hunk 3]
```diff
@@ -1258,12 +1258,22 @@ \section{Conclusion}
...
+institutions that make it possible that astronomers and programmers on
+their staff can contribute (part of) their time to the development of
+\astropy projects. Furthermore, the \astropy packages would not exisit
```
Comment (adrn) [inline main.tex]: exisit -> exist

File: main.tex [hunk 4]
```diff
@@ -1258,12 +1258,22 @@ \section{Conclusion}
...
+in their current form without a number of webservices for code
```
Comment (adrn) [inline main.tex]: Is "webservice" one word, or should be "web services"?

File: main.tex [hunk 5]
```diff
@@ -1258,12 +1258,22 @@ \section{Conclusion}
...
+hosting, continuous integration, and documentation; in particular,
+\astropy heavily relies on GitHub, Travis CI, Appveyor, CircleCI, and
+Read the Docs.  \astropypkg integrates with the SIMBAD database,
```
Comment (adrn) [inline main.tex]: integrates -> interfaces

**Review by adrn — CHANGES_REQUESTED** [review]
Some minor comments, otherwise looks good - thanks!

File: main.tex [hunk 6]
```diff
@@ -1258,12 +1258,22 @@ \section{Conclusion}
...
-
-We thank the many web services that make developing and maintaining this package
-feasible, such as GitHub, Travis CI, Appveyor, CircleCI, and Read the Docs.
+\acknowledgments The \astropy community is supported by and makes use
+of a number of organizations and services outside the traditional
+academic community. We thank Google for financing and organizing the
+Google Summer of Code (GSoC) program, that has funded severals
+students per year to work on \astropy related projects over the
+summer. These students ofter turn into long-term contributors. We also
+thank NumFOCUS and the Python Software Foundation for financial
```
Comment (crawfordsm) [inline main.tex]: Fairly trivial, but my feeling is this should be mentioned before summer of code

**Comment by crawfordsm** [comment]
Just as a note, @eteq and I agreed that we would list organizations that have specifically supported people to work on astropy.

File: main.tex (hunk 6 above)
Comment (hamogu) [reply main.tex]: I sorted in rough order of funding. I don't know exactly got much we got from NumFOCUS and the PSF, but we've had GSOC students for a few years not with 5-10 students per year. Let's say 5 years (not sure) * (5 students * 5000 $ + mentor summit + org payments) > 150 k$.

**Comment by crawfordsm** [comment]
How is it integrated with SIMBAD?  I imagine that should be part of a new paragraph included all the services that astropy does use like SOFA, etc that are not otherwise mentioned throughout the text

File: main.tex [hunk 7]
```diff
@@ -1258,12 +1258,22 @@ \section{Conclusion}
...
+support for various needs. Within the academic community, we thank
+institutions that make it possible that astronomers and programmers on
+their staff can contribute (part of) their time to the development of
+\astropy projects. Furthermore, the \astropy packages would not exisit
+in their current form without a number of webservices for code
+hosting, continuous integration, and documentation; in particular,
+\astropy heavily relies on GitHub, Travis CI, Appveyor, CircleCI, and
```
Comment (crawfordsm) [inline main.tex]: Perhaps add links to the websites here if they are not linked or cited elsewhere?

**Comment by bsipocz** [comment]
I would also drop simbad, the right way to mention that is in coordinate section. Otherwise we should really list everything we use in extern/cextern.

**Comment by hamogu** [comment]
@crawfordsm : The authorship sign-up form has filed to put in acknowledgements and we can consolidate. That will probably include STScI (several people), CfA (T. Aldcroft) and maybe afew others I'm not aware of. I was thinking or institutions were the contributions is more informal, e.g. in my situation I work on astropy sometimes in my "science" time, sometimes as part of a data analysis project that is e.g. HST or Chandra funded. I'm not going to list all grant numbers I've ever had where code was written tat may have ended up in astropy later. I'm sure I'm not the only one in this situation.

**Comment by hamogu** [comment]
Simbad is for coordinates _from_name. We should list SOFA, too. I forgot about that.

**Comment by hamogu** [comment]
I don't know if there is an offical acknowledgement sentence that's required for cfitsio and wcslib, but I can check what they say on their website. I don't think expat requires that.

**Comment by hamogu** [comment]
I don't think that SIMBAD is important enough to be mentioned in the coordinates section. It's just one function that calls the object resolver. Yet, part of the motivation of this paper is to give proper credit and thus I think it's important that we also give proper credit for those services that we make use of and there in particular those that depend on funding for research agencies themselves. And in my view, the acknowledgements section is there to acknowledge these things in the same way that I would acknowledge and HST grant for an observational paper in the acknowledgement section, but not discuss it in the main paper.

**Comment by bsipocz** [comment]
@hamogu - I feel we should rather cite all of these inline in the relevant sections.

**Comment by bsipocz** [comment]
OK. I can live with that. It just felt disproportionate compared to all the rest in the ack. exactly as we only use it in a minor method.

File: main.tex [hunk 8]
```diff
@@ -1258,12 +1258,22 @@ \section{Conclusion}
...
-feasible, such as GitHub, Travis CI, Appveyor, CircleCI, and Read the Docs.
+\acknowledgments The \astropy community is supported by and makes use
+of a number of organizations and services outside the traditional
+academic community. We thank Google for financing and organizing the
+Google Summer of Code (GSoC) program, that has funded severals
+students per year to work on \astropy related projects over the
+summer. These students ofter turn into long-term contributors. We also
+thank NumFOCUS and the Python Software Foundation for financial
+support for various needs. Within the academic community, we thank
+institutions that make it possible that astronomers and programmers on
```
Comment (pllim) [inline main.tex]: programmers -> developers *or* engineers (?)

File: main.tex [hunk 9]
```diff
@@ -1258,12 +1258,22 @@ \section{Conclusion}
...
+their staff can contribute (part of) their time to the development of
```
Comment (pllim) [inline main.tex]: Is "(part of)" really necessary?

File: main.tex (hunk 8 above)
Comment (eteq) [reply main.tex]: 👍  to "developers", which I think is the best generic term

File: main.tex (hunk 9 above)
Comment (eteq) [reply main.tex]: I agree to drop "part of".

**Review by eteq — COMMENTED** [review]
Modulo the various comments I think this is good, but one additional thought: perhaps the various "sets" of acknowledgements should be in different paragraphs?  In particular I think it's important the acknowledgement of e.g. the software ecosystem be separate form the acknowledgement of institutions.

**Review by adrn — APPROVED** [review]

File: main.tex [hunk 10]
```diff
@@ -1258,18 +1258,35 @@ \section{Conclusion}
...
-feasible, such as GitHub, Travis CI, Appveyor, CircleCI, and Read the Docs.
+\acknowledgments The \astropy community is supported by and makes use
+of a number of organizations and services outside the traditional
+academic community. We thank Google for financing and organizing the
+Google Summer of Code (GSoC) program, that has funded severals
+students per year to work on \astropy related projects over the
+summer. These students often turn into long-term contributors. We also
+thank NumFOCOS and the Python Software Foundation for financial
+support. Within the academic community, we thank
+institutions that make it possible that developers on
```
Comment (bsipocz) [inline main.tex]: sorry to be nitpicking on this, but I think it was better in the previous round to also mention astronomers (...that astronomers and developers on their staff...). This current wording can falsely suggest that it's only people in the engineering departments who fall in this category, while the reality is that some institutions are indeed OK with their staff using their **science** time to write community software.

**Comment by adrn** [comment]
Fixed a typo, but looks good otherwise.

**Pull request merged**